kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataPanel.js

```diff
@@ -396,11 +398,11 @@ define([
             var user = Jupyter.narrative.userId;
 
             if (!user) {
```

Comment:
Wow, this never got updated when we moved to the new Auth platform. It tries to parse an old style auth token instead. You can safely remove this whole if statement.

Reply:
Er, the `if (!user)` block starting at line ~400